google-cloud-storage/src/test/java/com/google/cloud/storage/StorageImplTest.java

```diff
@@ -384,7 +419,8 @@ public long millisTime() {
   private Bucket expectedBucket1, expectedBucket2, expectedBucket3;
 
   @BeforeClass
-  public static void beforeClass() throws NoSuchAlgorithmException, InvalidKeySpecException {
+  public static void beforeClass()
```

Comment:
`beforeClass()` => `setUp()`

Reply:
it's already there see
https://github.com/googleapis/java-storage/blob/master/google-cloud-storage/src/test/java/com/google/cloud/storage/StorageImplTest.java#L398-L411